.github/workflows/pull-request.yml

```diff
@@ -41,28 +46,47 @@ jobs:
           msg: |
             ## :white_check_mark: No Breaking Changes
           check_for_duplicate_msg: false
-  protoc:
+
+  java:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup Java
-        uses: actions/setup-java@v1
+
+      - uses: actions/setup-java@v1
         with:
           java-version: 11
-      - name: Setup Node
-        uses: actions/setup-node@v1
+          java-package: jdk
+
+      - name: setup protoc
+        run: ./install_proto.sh
+
+      - name: make
+        run: make PROTOC=./bin/protoc LANGUAGE=java OUTPUT=./java/src/main/java
+
+      - name: build
+        run: ./gradlew clean build
+        working-directory: java
+
+  node:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
         with:
           node-version: "12.x"
+
       - name: npm install
         run: npm install
         working-directory: node
-      - name: Setup Proto
+
+      - name: setup protoc
         run: ./install_proto.sh
-      - name: protoc via make
-        run: make PROTOC_BIN=./bin/protoc all
-      - name: Run java checks
-        run: ./gradlew clean build
-        working-directory: java
-      - name: Run node checks
-        run: npm test
-        working-directory: node
+
+      - name: make
+        run: make PROTOC=./bin/protoc LANGUAGE=node OUTPUT=./node
+
+# do we need to test protoc?
```

Comment:
+1 von mir. Ist nicht teuer. Zwischen Merge und Release können auch Tage/Wochen liegen. Gibt mir immer ein besseres Gefühl.

Reply:
+1 auch von mir. Hat uns in der Vergangenheit oft den Arsch gerettet.